### PR TITLE
added homepage

### PR DESCRIPTION
**Pull request by tusharkandpal**
- main page html based on figma design
- used wings-ui library & few own styles to override

Recording 2022-02-12 at 16 45 27

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Looks good to me, it would be nice if you have a preview

**Comment by tusharkandpal** [comment]
> Looks good to me, it would be nice if you have a preview

ya forgot it...adding now

File: index.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* ------------------------|| IMPORTS -----------------------*/
+@import url('./utilities/main.css');
+
+/*--------------------- || HOMEPAGE ----------------------------*/
+
+/* --------------------------- || CATEGORY SECTION ----------------- */
+.main {
+    padding: 2rem 7rem;
+    background-color: #F7F7F7;
```
Comment (shaksham08) [inline index.css:9]: you can use color variables instead of hardcoded colors

**Review by shaksham08 — APPROVED** [review]
Looks good to me.
Just added one comment

File: index.css (hunk 1 above)
Comment (tusharkandpal) [reply index.css:9]: i will create color variables at last.

**Pull request merged**